widgetastic_manageiq.py

```diff
@@ -1205,15 +1206,15 @@ class ItemsToolBarViewSelector(View):
         view_selector.selected
     """
     ROOT = './/div[contains(@class, "toolbar-pf-view-selector")]'
-    grid_button = Button(title='Grid View')
```

Comment:
REQUIRED: there is one more view with such buttons - DetailsToolBarViewSelector. It waits for your attention as well.